frontend/next.config.js

```diff
@@ -61,11 +62,17 @@ const moduleExports = {
       locales: ['en-US'],
       defaultLocale: 'en-US',
    },
+   ...(!SENTRY_AUTH_TOKEN && {
+      sentry: {
+         disableServerWebpackPlugin: true,
+         disableClientWebpackPlugin: true,
+      },
+   }),
 };
 
 // Make sure adding Sentry options is the last code to run before exporting, to
 // ensure that your source maps include changes from all other Webpack plugins
 module.exports = withSentryConfig(
    withTM(moduleExports),
-   sentryWebpackPluginOptions
+   SENTRY_AUTH_TOKEN ? sentryWebpackPluginOptions : undefined
```

Comment:
Forgot to include this in the commit message, but I followed steps here to figure this out:
https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#configure-source-maps